### PR TITLE
More disco

### PR DESCRIPTION
**Pull request by tharvik**
* add prez and demo link
* update disco desc

### CODE REVIEW

File: data/MLO/projects.yaml [hunk 1]
```diff
@@ -272,13 +272,29 @@ projects:
     applications:
       - Info
     description: Decentralized Collaborative Machine Learning
+    layman_desc: >
+      A collection of objects and routines to help you develop your own distributed
+      machine learning algorithm. It has two modes, federated, where a server is
+      keeping track of the model, and decentralized, where each participants has their
```
Comment (ineiti) [inline data/MLO/projects.yaml:278]:
```suggestion
      keeping track of the model, and decentralized, where each participant has their
```

**Review by ineiti — CHANGES_REQUESTED** [review]
Can you add a paragraph where you compare it to other solutions? What are the standalone features that make it special?

**Comment by tharvik** [comment]
> Can you add a paragraph where you compare it to other solutions? What are the standalone features that make it special?

I added
> Compared to other solutions, Disco is bridging the gap between people creating
      machine learning models and people having relevant data. By running without
      installation nor complex configuration, anyone can help. It's also a breeding
      ground for new machine learning technologies.

If you find something else, please tell me :sweat_smile:

File: views/products/presentation/disco.tpl [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  It runs in a browser, allowing it to work on pretty much any platform, simply
+  open the link in a browser and you're using it.
+  <br>
+  It allows creating your own machine learning tasks so that others can help
+  with theses by just loading their data in the browser.
+  By computing theses tasks, you create a machine learning model, a mathematical
+  system which allows a program to make predictions based on a given set of data.
+  It fits many use cases, such as determining the likeliness that some hospital
+  patient has a disease, or automatically determine what an image represents.
+</p>
```
Comment (ineiti) [inline views/products/presentation/disco.tpl:18]: Copy the list of the papers Disco (will be / is) based on from the github page:

https://github.com/epfml/disco/blame/develop/README.md#L21

And describe which algorithms are already implemented, and which will (probably) be.

**Review by ineiti — CHANGES_REQUESTED** [review]

File: views/products/presentation/disco.tpl (hunk 2 above)
Comment (tharvik) [reply views/products/presentation/disco.tpl:22]: Papers added

**Review by ineiti — APPROVED** [review]

**Pull request merged**